src/profile-logic/summary-worker/index.js

```diff
@@ -1,32 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-import { createStore, applyMiddleware } from 'redux';
-import threadDispatcher from '../../utils/thread-middleware';
-import handleMessages from '../../utils/message-handler';
```

Comment:
message-handler isn't used anymore, you can remove it as well. We know where to find it if we need it.